gammapy/stats/data.py

```diff
@@ -63,7 +63,9 @@ def __str__(self):
         keys = ['n_on', 'n_off', 'a_on', 'a_off',
                 'alpha', 'background', 'excess']
         values = [self.n_on, self.n_off, self.a_on, self.a_off,
-                  self.alpha, self.background(), self.excess()]
+#                  self.alpha, self.background(), self.excess()]
+                  ### JLK
```

Comment:
Thanks for fixing this. But please remove the `### JLK` line.
We use git history to see who did what and never leave such comments in the code.